apps/judicial-system/web/messages/InvestigationCases/Prosecutor/caseFiles.ts

```diff
@@ -8,6 +8,15 @@ export const icCaseFiles = {
       'Notaður sem titill á rannsóknargagna skrefi í rannsóknarheimildum.',
   }),
   sections: {
+    parentCaseFiles: defineMessages({
+      heading: {
+        id:
+          'judicial.system.investigation_cases:case_files.parentCaseFiles.heading',
```

Comment:
We usually write the ids in_this_format. Please consider changing parentCaseFiles to parent_case_files